boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160.yaml

```diff
@@ -7,7 +7,7 @@ toolchain:
   - xtools
   - zephyr
 ram: 88
-flash: 256
+flash: 900
```

Comment:
Here indeed, the linker reports 1 MB of flash, so this value can be increased (although that 900 kB might be confusing).
But for the other values, I see that linker reports exactly the values that are currently provided in the `.yaml` files:
- 192 kB of RAM and 192 kB of flash for `nrf5340dk_nrf5340_cpuapp`
- 128 kB of RAM and 192 kB of flash for `nrf9160dk_nrf9160_ns`

I checked it on the recent `main` (4f75848e71b136ad23e06bb0b7dade1cf61bbf68). Are you sure you used unmodified upstream Zephyr?

Reply:
I must have made the mistake you suspect, just confirmed what you are seeing.

Sorry about that.

Will update so it's just this patch left.

Reply:
fixed